packages/gallery/src/components/gallery/proGallery/galleryContainerExtraNew.js

```diff
@@ -519,6 +517,12 @@ export class GalleryContainer extends React.Component {
     return can;
   }
 
+  allowSSROpacity() {
+    const {settings = {}} = this.props;
+    const {disableSSROpacity = false} = settings;
+    return !!utils.isSSR() && isPrerenderMode() && !disableSSROpacity;
```

Comment:
didn't we remove the isSSR in the core?
the prerenderMode is the only condition.

Also, why is the `settings.disableSSRopacity` needed?

Reply:
disableSSRopacity is needed for testing the SSR - when using ?ssrWarmupOnly=true the gallery will load with opacity 0 (if no custom element - thunderbolt), so we need to add ?ssrWarmupOnly=true&disableSSRopacity=true opacity will be 1.

Reply:
the opacity is needed only on SSR (not on prerenderMode), maybe I can use setting.enableSsrOpacity from the wrapper and (instead of disableSSRopacity)